src/test/java/io/crate/testing/ClusterTest.java

```diff
@@ -39,7 +40,7 @@
 public class ClusterTest extends BaseTest {
 
     private static final String CLUSTER_NAME = "cluster";
-    private static final String VERSION = "0.57.1";
+    private static final String VERSION = "3.0.3";
```

Comment:
maybe it makes sense to make this public and use it everywhere? (or move it to a kind of config class)